.gitignore

```diff
@@ -63,10 +63,7 @@ db/schema.rb
 db/fixtures/ae_datastore/ManageIQ/.manifest.yaml
 
 # log/
-log/*.gz
-log/*.log
-log/*.txt
-log/call_graph_*.html
+log
```

Comment:
Can you put a leading `/` here so we only ignore the top-level log dir and not any future subdir?

Reply:
Done!

Reply:
leading or trailing?

Reply:
@chessbyte I just went with what he said, but can fix it as needed.

Reply:
Leading is correct, we only want to ignore the top level log dir